Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all Marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if ($mark->isRunning()) {
+                $runningMarks[] = $mark;
+            }
+        }
+
+        // short loop = reduce delta between pauses
+        // same loop as in resume()
+        foreach ($runningMarks as $mark) {
+            $mark->pause(true);
```

Comment:
May be we should set all parameters in a Library file? 
It keeps the behaviors, even if the optional parameter is changed (cf native PHP functions). 
Moreover, the (human) reader will know all the parameters without seeing the method. 
Technically I don't know the difference in PHP. (Some other languages are slower while optional parameters are not overloaded).